action.yml

```diff
@@ -21,6 +21,8 @@ inputs:
   password:
     description: 'The password to authenticate with the ORT Server instance'
     required: false
+
```

Comment:
Commit message: Maybe link to https://www.jetbrains.com/help/idea/using-language-injections.html#use-language-injection-comments for reference?